backends-clickhouse/src/main/scala/io/glutenproject/utils/CHExpressionUtil.scala

```diff
@@ -41,7 +41,9 @@ object CHExpressionUtil {
     SPLIT_PART -> Set(EMPTY_TYPE),
     TO_UNIX_TIMESTAMP -> Set(DATE_TYPE),
     UNIX_TIMESTAMP -> Set(DATE_TYPE),
-    MIGHT_CONTAIN -> Set(EMPTY_TYPE)
+    MIGHT_CONTAIN -> Set(EMPTY_TYPE),
```

Comment:
@taiyang-li Please help to review. Thanks.

Reply:
LGTM